test/models/pet_test.rb

```diff
@@ -83,6 +83,28 @@ class PetTest < ActiveSupport::TestCase
         assert_not Pet.with_photo.include?(pet_without_image)
       end
     end
+
+    context ".unique_species" do
+      should "returns an array of unique species" do
+        create(:pet, species: "Dog", placement_type: "Adoptable")
+        create(:pet, species: "Cat", placement_type: "Adoptable")
+        create(:pet, species: "Dog", placement_type: "Adoptable")
+
+        unique_species = Pet.unique_species
+
+        assert_equal ["Cat", "Dog"], unique_species.sort
+      end
+
```

Comment:
We should also add a test here that handles the case where the pet is adopted i.e., has a `match` with `match_type` of `adoption`